hooks/pre-push

```diff
@@ -1,2 +1,11 @@
 #!/bin/bash -e
-for f in $0.d/*; do $f; done
+rust=0
+for file in $(git diff-index --name-only HEAD); do
+    if [[ -n "$TRAVIS" ]] || [[ $file == *.rs ]] || [[ $file == *.toml ]]; then
```

Comment:
`[[ $file == *.toml ]]`

That's the first I've ever seen this syntax. I assume it checks that the file ends with `.toml`.

Reply:
Yes, it does. I suppose we could change it to explicitly check for `Cargo.toml` if we wanted.